datadog_checks_dev/pyproject.toml

```diff
@@ -66,7 +66,10 @@ cli = [
     "click~=8.1.6",
     "codespell",
     "colorama",
-    "datamodel-code-generator~=0.21.1",
+    # We need the following PRs to address Pydantic deprecations:
+    # https://github.com/koxudaxi/datamodel-code-generator/pull/1468
+    # https://github.com/koxudaxi/datamodel-code-generator/pull/1485
```

Comment:
nit - I think the changelog entry is enough and I would add this comment to the PR description and not in the code itself since it won't be very useful after the next bump imo

Reply:
hmm, your point about the next bump making this irrelevant is well taken, I will move this to the PR description.